extensions/sniffer/package.json

```
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "sniffer",
+  "title": "Hotspot",
```

```suggestion
  "title": "Sniffer",
```

Should we use the same wording?